doc/dev/CredScan-Process.md

```diff
@@ -0,0 +1,70 @@
+# Guide for monitoring CredScan checks
+
+This guide describes how package owners can monitor their package's Credential Scanner (CredScan) status and correct
+any warnings.
+
+General information about CredScan can be found in the overview documentation at [aka.ms/credscan][credscan_doc]. The
+Azure SDK's motivation and methodology for running CredScan is documented [here][devops_doc].
+
+## Table of Contents
+
+- [Check CredScan status](#check-credscan-status)
+- [Correct active warnings](#correct-active-warnings)
+    - [True positives](#true-positives)
+    - [False positives](#false-positives)
+
+## Check CredScan status
+
+CredScan is run each night over the entire `azure-sdk-for-java` repository as part of the
+[java - aggregate-reports][aggregate_reports] pipeline. The scan produces a list of active warnings in the "Post
+Analysis" task of the "ComplianceTools" job ([example output][credscan_output]).
+
+Each warning will begin with the path to the file containing a potential credential, as well as the row and column where
+the credential string begins. For example, for a potential credential that starts in row 3 and column 20 of a
+particular file:
```

Comment:
Is there a way we could proactively notify the team/codeowners about this rather than everyone checking the results of the aggregate report pipeline?

Reply:
@sima-zhu is working on a template to enable this in PR validation.